pkg/network/node/ovscontroller.go

```diff
@@ -977,6 +977,8 @@ func (oc *ovsController) SetNamespaceEgressViaEgressIPs(vnid uint32, egressIPsMe
 		// Namespace wants egressIP, but no node hosts it, so drop
 		otx.AddFlow("table=101, priority=100, reg0=%d, actions=drop", vnid)
 	} else {
+		// check if there is a connection already stored in conntrack
+		otx.AddFlow("table=101, priority=101, reg0=%d, ct_state=+est, actions=move:NXM_NX_CT_NW_DST[]->NXM_NX_TUN_IPV4_DST[],output:vxlan0", vnid)
```

Comment:
So I had suggested this, but after thinking some more, I think it's wrong; `ct_nw_dst` will be the `nw_dst` of the original connection (ie, the external IP we're using the egress IP to reach), whereas we want the `tunnel_dst` of the original connection, which doesn't seem to be stored in conntrack...

I think we could store it in `ct_mark` though... we'd have to copy it there explicitly when sending the first packet, and then copy it back out when sending later packets.

Reply:
done